src/main/java/frc/robot/commands/ArmAutoCommand.java

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.ArmSubsystem;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import java.lang.Math;
+
+public class ArmAutoCommand extends CommandBase {
+  private static final double firstArmLength = 110;
+  private static final double secondArmLength = 94;
```

Comment:
Use the constants from the subsystem. These shouldn't be here at all.

Reply:
done, now same as constants